src/content/test/pointer-motion/target-size.tsx

```diff
@@ -12,16 +12,17 @@ export const infoAndExamples = create(({ Markup }) => (
         <p>
             When touch targets are too small or too close together, it becomes difficult for users to activate them and having sufficient
             target spacing helps all users who may have difficulty in confidently targeting or operating small controls. Users who benefit
-            include, but are not limited to people who use a mobile device where the touch screen is the primary mode of interaction, or the
-            use of mouse, stylus or touch input for people with mobility impairments such as hand tremors.
+            include, but are not limited to, people who use a mobile device where the touch screen is the primary mode of interaction, or
+            that use of mouse, stylus or touch input for people with mobility impairments such as hand tremors.
         </p>
 
         <h2>How to fix</h2>
         <p>
             All touch targets must be at least 24 by 24 CSS pixels in size, where size is computed by taking the largest unobscured area of
-            the touch target. If the size of the target is insufficient, then it must be at least 24 CSS pixels away from any other touch
-            target. All exception criteria may be used to achieve success for this test for example, providing a different control on the
-            same page that meets target size.
+            the touch target. A touch target may consist of a single actionable element, or an actionable element along with its clickable
+            label or other programmatically associated clickable element. If the size of the target is insufficient, then it must be at
+            least 24 CSS pixels away from any other touch target. All exception criteria may be used to achieve success for this test for
```

Comment:
This feels like it needs to be split up. Maybe something like

```suggestion
            least 24 CSS pixels away from any other touch target. All exception criteria may be used to achieve success for this test, for
```